runner.js

```diff
@@ -30,14 +30,13 @@ const GENESIS_PATH = `data/${rb(4).toString('hex')}.json`;
 
 const env = Object.assign({DELEGATES: num, GENESIS_PATH}, process.env);
 
-let delegates = [];
-let producers = [];
+const delegates = [];
+const producers = [];
+const bank      = Account();
 
 const genesis = Genesis();
 
-// const account = Account();
-// producers.push(account);
-// genesis.addProducer(account.address.toString('hex'), 1);
+genesis.addAccount(bank.address.toString('hex'), 1000000);
```

Comment:
Look at changes in core/account https://github.com/WingsDao/griffin-consensus-poc/pull/11/commits/ba63d99a5dc8232ee66ebd7d13e3f60682098e0a#diff-9d53720ebbf552f679e0fd32f4094f9fR87 , you can use new method to get address in string representation.

Reply:
Nope. I can't. Genesis doesn't support 0x-prefixed addresses.